src/include/atomic.h

```diff
@@ -70,6 +70,17 @@ namespace ceph {
       ceph_spin_unlock(&lock);
       return ret;
     }
+    int cas(T o, T n) {
```

Comment:
I agree true/falue return is best.  So just switch the return type to bool to make that clear?  (It couldn't tell if that's what the AO_compare_and_swap op was returning)

Or did I misunderstand?